src/components/Auth/AuthModal.css

```diff
@@ -0,0 +1,42 @@
+.modal {
```

Comment:
I'd avoid the !important flags, this is modifying bootstrap core modal styling. Either append another class to make your modal more specific or use css modules